evergreen/config.yml

```diff
@@ -1398,8 +1435,28 @@ buildvariants:
   tasks:
   - name: fuzzer-tests
 
-- name: ubuntu2004-network
-  display_name: "Ubuntu 20.04 x86_64 (Clang 11 Baas network tests)"
+- name: ubuntu2004-network-nonideal
+  display_name: "Ubuntu 20.04 x86_64 (Utunbu2004 Baas network tests - nonideal transfer)"
+  run_on: ubuntu2004-large
+  expansions:
+    clang_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/clang%2Bllvm-11.0.0-x86_64-linux-gnu-ubuntu-20.04.tar.xz"
+    cmake_url: "https://s3.amazonaws.com/static.realm.io/evergreen-assets/cmake-3.20.3-linux-x86_64.tar.gz"
+    cmake_bindir: "./cmake_binaries/bin"
+    fetch_missing_dependencies: On
+    c_compiler: "./clang_binaries/bin/clang"
+    cxx_compiler: "./clang_binaries/bin/clang++"
+    cmake_build_type: RelWithDebInfo
+    run_with_encryption: On
+    baas_admin_port: 9098
+    test_logging_level: trace
```

Comment:
are we sure we want trace here? It is indeed useful when we catch bugs though.

Reply:
The trace is for extra debugging, in case there are failures - it may be adjusted at a later point once these tests are stable.